src/main/java/org/benetech/servicenet/service/impl/ActivityServiceImpl.java

```diff
@@ -93,6 +97,26 @@ public ActivityServiceImpl(ActivityRepository activityRepository, RecordsService
         }
     }
 
+    @Override
+    @Transactional(readOnly = true)
+    public List<ActivityRecordDTO> getPartnerActivitiesByOrganizationId(UUID orgId) {
+        log.debug("Getting partner Activity Records for organization: {}", orgId);
+        organizationMatchService.findAllNotHiddenForOrganization(orgId);
```

Comment:
What's the point of doing it twice? Here and in return in next line. Also do we need logging here?